packages/atomic/cypress/utils/setupComponent.ts

```diff
@@ -29,20 +36,18 @@ export function setupIntercept() {
 
 export function setUpPage(htmlCode: string) {
   setupIntercept();
-  // Setup page with new component
   cy.visit('http://localhost:3333/pages/test.html');
   cy.injectAxe();
   injectComponent(htmlCode);
-  cy.wait(1000);
+  cy.wait(500); // TODO: waiting for @coveoSearch is less flaky
 }
 
 export function setUpPageNoSearch(htmlCode: string) {
   setupIntercept();
-  // Setup page with new component
-  cy.visit('http://localhost:3333/pages/test-no-search.html');
+  cy.visit('http://localhost:3333/pages/test.html');
   cy.injectAxe();
-  injectComponent(htmlCode);
-  cy.wait(1000);
+  injectComponent(htmlCode, false);
+  cy.wait(300);
```

Comment:
1000 was just exaggerated

Reply:
in this case, we already inject & execute the search, do you think we still need a wait here?

Reply:
Here there is no search executed, I give a small delay to make sure everything is rendered correctly, flakiness prevention :D